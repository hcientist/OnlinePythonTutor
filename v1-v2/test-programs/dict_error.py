def foo():
  local_y[('tup', 'le')] = set([1, 2, 3])

foo()
