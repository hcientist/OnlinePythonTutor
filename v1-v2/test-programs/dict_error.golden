[ { "event" : "step_line",
    "func_name" : "<module>",
    "globals" : {  },
    "line" : 1,
    "stack_locals" : [  ],
    "stdout" : ""
  },
  { "event" : "step_line",
    "func_name" : "<module>",
    "globals" : { "foo" : [ "function",
            99999,
            "<function foo at 0x4b8970>"
          ] },
    "line" : 4,
    "stack_locals" : [  ],
    "stdout" : ""
  },
  { "event" : "call",
    "func_name" : "foo",
    "globals" : { "foo" : [ "function",
            99999,
            "<function foo at 0x4b8970>"
          ] },
    "line" : 1,
    "stack_locals" : [ [ "foo",
          {  }
        ] ],
    "stdout" : ""
  },
  { "event" : "step_line",
    "func_name" : "foo",
    "globals" : { "foo" : [ "function",
            99999,
            "<function foo at 0x4b8970>"
          ] },
    "line" : 2,
    "stack_locals" : [ [ "foo",
          {  }
        ] ],
    "stdout" : ""
  },
  { "event" : "exception",
    "exception_msg" : "NameError: global name 'local_y' is not defined",
    "func_name" : "foo",
    "globals" : { "foo" : [ "function",
            99999,
            "<function foo at 0x4b8970>"
          ] },
    "line" : 2,
    "stack_locals" : [ [ "foo",
          {  }
        ] ],
    "stdout" : ""
  },
  { "event" : "return",
    "func_name" : "foo",
    "globals" : { "foo" : [ "function",
            99999,
            "<function foo at 0x4b8970>"
          ] },
    "line" : 2,
    "stack_locals" : [ [ "foo",
          { "__return__" : null }
        ] ],
    "stdout" : ""
  },
  { "event" : "exception",
    "exception_msg" : "NameError: global name 'local_y' is not defined",
    "func_name" : "<module>",
    "globals" : { "foo" : [ "function",
            99999,
            "<function foo at 0x4b8970>"
          ] },
    "line" : 4,
    "stack_locals" : [  ],
    "stdout" : ""
  }
]
